### PR TITLE
Upgrade to Scala 2.13.0-RC1

### PR DESCRIPTION
**Pull request by olafurpg**
Currently blocked by portable-scala.

### CODE REVIEW

**Comment by Philippus** [comment]
portable-scala has been released for 2.13.0-RC1 now

File: utest/jvm/src/test/scala-2.12/test/utest/FormatterTests.scala [hunk 1]
```diff
@@ -9,7 +9,10 @@ import concurrent.ExecutionContext.Implicits.global
 object FormatterTests extends utest.TestSuite {
   def trim(s: String): String = {
     // Predef.augmentString = work around scala/bug#11125
-    Predef.augmentString(s.trim).lines.map(_.reverse.dropWhile(_ == ' ').reverse).mkString("\n").replaceAll(" \\d+ms", "")
+    Predef.augmentString(s.trim).lines.map(_.reverse.dropWhile(_ == ' ').reverse)
+      .mkString("\n")
+      .replaceAll(" \\d+ms", "")
+      .replaceAll(":\\d+\\)", ":\\)")
```
Comment (olafurpg) [inline utest/jvm/src/test/scala-2.12/test/utest/FormatterTests.scala]: the line numbers in scala-library changed in 2.12.8 compared to 2.12.6 so now we strip them out from stack traces

File: build.sbt [hunk 2]
```diff
@@ -25,9 +26,16 @@ lazy val utest = crossProject(JSPlatform, JVMPlatform, NativePlatform)
     }),
 
     unmanagedSourceDirectories in Compile += {
-      val v = if (scalaVersion.value startsWith "2.10.") "scala-2.10" else "scala-2.11"
+      val v = "scala-" + scalaVersion.value.split("\\.").take(2).mkString(".")
```
Comment (olafurpg) [inline build.sbt]: This was needed to make `src/main/scala-2.13` work with 2.13.0-RC1, which we use for `MacroCompat.scala`

**Comment by olafurpg** [comment]
I took the liberty to push a v0.6.7 tag, the only other PR since v0.6.6 is #186 which changes the macro code generation so I figured a new release would be better than reusing v0.6.6

https://travis-ci.org/lihaoyi/utest/builds/517264573

**Pull request merged**